### ISSUE TITLE
digicam-spe does not work: too slow fit_template() and find_pulse_2()

### ISSUE THREAD

**Issue by dneise** [cta-sst-1m/digicampipe#174]
after #172 was fixed I further tried to execute `digicampipe-spe --compute`. I let it run for a while but the progress bar did not move at all.

I put a few `print()` here and there and found that `fit_template()` really took forever. 
So I commented out `fit_template()` and commented in `compute_amplitude()`.
Now it worked .. but it was still very slow.

It measured more of the run times and found that `find_pulse_2()` took 2.2sec per event .. so at 10k events this takes ~22k seconds which is about 6hours ... pretty much exactly what @calispac reported. 

So I played a bit around and implemented a quicker pulse finder, aptly named `find_pulse_3()` .. 
it takes ~1ms per event.

**Comment by dneise** [comment]
In #175 I present this quicker version of a pulse finder. 

However, in order to find out if that pulse finder really works, I need to look at the charge spectra.
I planned to got on now with `digicampipe-spe --fit` but also this does not work :-(.

I will open another issue for this.
